bundle/manifests/external-secrets-operator-example_console.openshift.io_v1_consolequickstart.yaml

```diff
@@ -0,0 +1,164 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  annotations:
+    capability.openshift.io/name: Console
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: external-secrets-operator-example
+spec:
+  conclusion: You have successfully deployed a sample application with secrets managed
+    by the External Secrets Operator for Red Hat OpenShift!
+  description: Deploy simple examples of the External Secrets Operator for Red Hat
+    OpenShift CRDs.
+  displayName: External Secrets Operator for Red Hat OpenShift Example
+  durationMinutes: 10
+  introduction: |-
+    # External Secrets Operator for Red Hat OpenShift
+
+    The External Secrets Operator for Red Hat OpenShift enables you to bring secrets or other "confidential material" into an OpenShift cluster
+    from an external source such as CyberArk Conjur, HashiCorp Vault, or any of the cloud service provider secrets managers.
```

Comment:
nit: I would suggest using the providers qualified downstream like AWS, GCP, Vault.. at all places.

Reply:
Thanks for the suggestion, updated.